src/DebugEngineHost.VSCode/HostMarshal.cs

```diff
@@ -120,7 +121,7 @@ public static string GetDataBreakpointStringForIntPtr(IntPtr stringId)
         /// <returns>IntPtr to a BSTR which can be returned to VS.</returns>
         public static IntPtr GetIntPtrForDataBreakpointAddress(string address)
         {
-            throw new NotImplementedException();
+            return Marshal.StringToBSTR(address);
```

Comment:
>return [](http://example.com/codeflow?start=12&length=6)

You probably should have a matching free function. I am assuming right now we just leak the string.

Reply:
Actually, I guess this is a general bug we have in MIEngine.